convert-overrides.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
```

Comment:
Why is this file included? It doesn't seem necessary?

Reply:
Didn't realize it was even in the repo, must have accidently run git add -A when I had multiple files and missed it, removed now.